src/main/resources/interface-specification.yml

```
@@ -159,6 +159,132 @@ paths:
               schema:
                 type: integer
               description: Time interval in milliseconds. Allowed requests will be constantly replenished during the interval. At the end of the interval the max allowed requests will be available
+  '/purposes/eservices/{eserviceId}/consumer/{consumerId}':
```

not sure if there are specific requirements, but what about
```suggestion
  '/purposes':
```
 moving `eserviceId` and `consumerId` as optional query parameters?

---

> not sure if there are specific requirements, but what about
> 
> moving `eserviceId` and `consumerId` as optional query parameters?

I agree on passing those as query parameters but they have to be required

---

fine by me, the lift of this constraint in the future is not a real breaking change so I guess can be done when required.

Leaving to @beetlecrunch the last word about the change

---

Nothing against the change proposed.
As @AsterITA mentioned, these query parameters must be required.